english/candyland/diabetes.md

```diff
@@ -0,0 +1,19 @@
+You light a small fire in the middle of the marshmallow room.
```

Comment:
Yes, the link works. The problem is that if the reader starts at `language.md`, they will never reach this page (unless I am missing something). You need to add a link to the existing story that will take the reader to this page - otherwise most readers will never see your change.